<launch>
    <node pkg="rosbag" type="play" name="rosbag" required="true" args="-s 20 $(find camera_calibration)/../../../data/2016-11-22-14-32-13_test.bag"/>
    <node name="imag_rect" pkg="image_proc" type="image_proc">
        <remap from="/image_raw" to="/sensors/camera/image_color"/>
        <remap from="/camera_info" to="/sensors/camera/camera_info"/>
    </node>
    <node pkg="rqt_gui" type="rqt_gui" name="rqt_gui" args="--perspective-file $(find camera_calibration)/launch/play.perspective"/>
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find camera_calibration)/launch/rviz_play.rviz"/>

    <node pkg="camera_calibration" type="findPlaneInPC_node" name="findPlaneInPC"/>
    <node pkg="rqt_console" type="rqt_console" name="rqt_console"/>
    <node pkg="camera_calibration" type="findPlaneInImages_node" name="findPlaneInImages"/>

    <param name="filePath" type="string" value="/home/yousof/RideCellAssignment/res.txt" />
    <node pkg="camera_calibration" type="saver_node" name="saver" output="screen"/>

    <node pkg="camera_calibration" type="keyBoard_node" name="keyBoard"/>
</launch>

 <!--  output="screen" -->
